### PR TITLE
Use neutron instead of neutron-legacy by default during devstack inst…

### PR DESCRIPTION
**Pull request by animationzl**
Fix theopenlab/openlab#16

Move the neutron services in default local-conf instead of being optional.

### CODE REVIEW

**Comment by animationzl** [comment]
For the change 'q-fwaas-v* -> neutron-fwaas-v*' refer to these
http://git.openstack.org/cgit/openstack/neutron-fwaas/tree/devstack/plugin.sh#n78
http://git.openstack.org/cgit/openstack/neutron-fwaas/tree/devstack/plugin.sh#n95

**Comment by animationzl** [comment]
For the deletion of q-lbaasv2, will enable neutron-lbaasv2 by default.
http://git.openstack.org/cgit/openstack/neutron-lbaas/tree/devstack/settings#n8

File: roles/create-devstack-local-conf/tasks/main.yml [hunk 1]
```diff
@@ -120,26 +121,11 @@
...
   when:
-    - '"fwaas-v2" in enable_services'
-
-- name: create devstack local conf with neutron enabled instead of neutron-legacy
-  shell:
-    cmd: |
-      set -e
-      set -x
-      cat << EOF >> /tmp/dg-local.conf
-      disable_service q-agt,q-svc,q-dhcp,q-l3,q-meta,q-metering
```
Comment (kiwik) [inline roles/create-devstack-local-conf/tasks/main.yml]: Please add disable_service q-* into new patch.

File: roles/create-devstack-local-conf/tasks/main.yml [hunk 2]
```diff
@@ -120,26 +121,11 @@
...
-      enable_service neutron-agent,neutron-api,neutron-dhcp,neutron-l3,neutron-metadata-agent,neutron-metering
-      EOF
-    executable: /bin/bash
-    chdir: '{{ ansible_user_dir }}/workspace'
-  environment: '{{ zuul | zuul_legacy_vars }}'
-  when:
-    - '"neutron" in enable_services'
+    - '"fwaas-v2" in enable_services'
```
Comment (kiwik) [inline roles/create-devstack-local-conf/tasks/main.yml]: Please add new line, other part code looks good to me.

**Pull request merged**